Makefile

```diff
@@ -236,7 +236,7 @@ container-docker: buildx # util target to build container images using docker bu
 	arch=$$(echo $(PLATFORM) | cut -d'/' -f2); \
 	echo "Building for $$os/$$arch"; \
 	docker buildx build \
-		$(ACTION) \
+		$(BUILDX_ACTION) \
```

Comment:
Add this in Contributing readme - that we can set `BUILDX_ACTION=--push` to push the image to CR?